### PR TITLE
Properly namespace `OC_Avatar`

### PR DESCRIPTION
**Pull request by LukasReschke**
@nickvergessen @th3fallen Easy one…

### CODE REVIEW

File: core/avatar/controller.php [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 namespace OC\Core\Avatar;
 
+use OC\Avatar;
+
 class Controller {
```
Comment (DeepDiver1975) [inline core/avatar/controller.php]: is that class touched by a unit test?

Comment (LukasReschke) [inline core/avatar/controller.php]: No. It's a horrible class that pretends to be a controller using the appframework but instead if complete static code …

**Comment by LukasReschke** [comment]
```
09:09:38 PHP Fatal error:  Class 'OC\DB\OCSqlitePlatform' not found in /var/jenkins/workspace/pull-request-analyser-ng-simple@2/label/SLAVE/lib/private/db/connectionfactory.php on line 99
```

Gotta love this bot… Mixes up pull requests now or what…?

**Comment by LukasReschke** [comment]
@owncloud-bot Retest this please.

File: core/avatar/controller.php (hunk 1 above)
Comment (DeepDiver1975) [inline core/avatar/controller.php]: can we change that? :speak_no_evil:

Comment (LukasReschke) [inline core/avatar/controller.php]: Sure. But not in that PR ;-)

File: tests/lib/avatar.php [hunk 2]
```diff
@@ -6,6 +6,9 @@
  * later.
  * See the COPYING-README file.
  */
+
+use OC\Avatar;
+
 class Test_Avatar extends \Test\TestCase {
```
Comment (nickvergessen) [inline tests/lib/avatar.php]: Namespace the test in the same PR? :see_no_evil:

**Comment by nickvergessen** [comment]
Looks good in general, although I'd love to see the test result pass first :p

**Comment by LukasReschke** [comment]
Let's close this in favour of https://github.com/owncloud/core/pull/14705.

**Pull request closed without merging**